lib/happo/public/happo-runner.js

```diff
@@ -6,11 +13,11 @@ window.happo = {
   currentRenderedElement: undefined,
   errors: [],
 
-  define: function(description, func, options) {
+  define: function define(description, func, options) {
```

Comment:
Are you sure that doesn't just work? The babel in-browser transpiler has surprised me before.